core/zip/src/RZip.cxx

```diff
@@ -130,24 +130,23 @@ void R__zipPrecisionCascade(int *srcsize, char *src, int *tgtsize, char **tgts,
       memset(irep,0,tgt_number*sizeof(int));
       return;
    }
-
    auto content = reinterpret_cast<ROOT::Internal::PrecisionCascadeConfigArrayContent*>(configarray);
-   (void) configsize;
```

Comment:
Since the assert is the only use of `configsize` and it is removed in Release build, this line prevent the compiler from complaining about unused variable/parameter.